File: 11_Container_With_Most_Water/src/test/java/SolutionTest.java
import org.junit.Assert;
import org.junit.Test;

public class SolutionTest {
    @Test
    public void testMaxArea() {
        int[] input = {1,8,6,2,5,4,8,3,7};
        Solution sol = new Solution();
        Assert.assertEquals(49, sol.maxArea(input));

    }
}

File: 20_Valid_Parentheses/src/test/kotlin/SolutionTest.kt
import io.kotlintest.shouldBe
import io.kotlintest.specs.FunSpec

class SolutionTest : FunSpec({
    val sol = Solution()
    test("basic test") {
        sol.isValid("()") shouldBe true
        sol.isValid("()[]{}") shouldBe true
        sol.isValid("(]") shouldBe false
        sol.isValid("([)]") shouldBe false
        sol.isValid("{[]}") shouldBe true
    }
    test("only close should be false") {
        sol.isValid(")") shouldBe false
        sol.isValid("]") shouldBe false
        sol.isValid("}") shouldBe false
    }
})
